apps/judicial-system/web/src/components/IndictmentCaseFilesList/IndictmentCaseFilesList.tsx

```diff
@@ -201,34 +237,34 @@ const IndictmentCaseFilesList: FC<Props> = ({
           ))}
         </Box>
       )}
-      {courtRecords?.length || rulings?.length ? (
+      {filteredFiles.courtRecords?.length || filteredFiles.rulings?.length ? (
         <Box marginBottom={5}>
           <Text variant="h4" as="h4" marginBottom={1}>
             {formatMessage(strings.rulingAndCourtRecordsTitle)}
           </Text>
-          {courtRecords && courtRecords.length > 0 && (
-            <RenderFiles caseFiles={courtRecords} onOpenFile={onOpen} />
-          )}
+          {filteredFiles.courtRecords &&
+            filteredFiles.courtRecords.length > 0 && (
+              <RenderFiles
+                caseFiles={filteredFiles.courtRecords}
+                onOpenFile={onOpen}
+              />
+            )}
```

Comment:
Nice! Did you consider also moving the rulings check below to the filePersmissions handling?

Reply:
Thanks! I've added a new permission for ruling files 🙌